src/bridge/cockpitpcpmetrics.c

```diff
@@ -410,6 +413,13 @@ cockpit_pcp_metrics_tick (CockpitMetrics *metrics,
     {
       cockpit_metrics_send_meta (metrics, meta);
       json_object_unref (meta);
+
+      /* We can't compute deltas across a meta message since
```

Comment:
This is a ```TODO:``` for now. We need to be able to compute deltas across meta messages. Otherwise the value point of doing deltas in the bridge is lost.